addons/web/static/src/js/chrome/action_manager.js

```diff
@@ -536,7 +536,7 @@ var ActionManager = Widget.extend({
                 var message = _t('A popup window has been blocked. You ' +
```

Comment:
@mart-e translations are handled correctly for string that are split this way?

Reply:
Looks like babel handles it fine
https://github.com/odoo/odoo/blob/master/addons/web/i18n/web.pot#L391

Reply:
Ok good to know, thank you for checking.